ncm-grub/src/main/perl/grub.pm

```diff
@@ -165,7 +165,7 @@ sub password
         return SUCCESS;
     } elsif (!$tree->{enabled}) {
         $self->info("removing grub password");
-        $grub_fh->remove_lines(qr/^password\s+/, '');
+        $grub_fh->remove_lines(qr/^password\s+/, q(no good line));
```

Comment:
something that will never match is better instead of the `no good liine`: use `qr/$./` (match a character after end of string)